package.json

```diff
@@ -12,6 +12,7 @@
   ],
   "license": "MIT",
   "dependencies": {
+    "lodash": "4.17.15",
```

Comment:
i suggest not to add further dependencies unless we must as we share this package with our customers.
lodash requires node ver > 4.0 and we can not guarentee this is the case for everyone uses this package.

you can replace _.includes with indexOf